src/sass/layouts/_header.scss

```
@@ -0,0 +1,72 @@
+header {
+  @include flex($justify: space-between, $align: center);
+  font-size: $fs-light;
+  @include pd($y: 1rem);
```

```suggestion
  @include flex($justify: space-between, $align: center);
  @include pd($y: 1rem);
  font-size: $fs-light;
```
đưa hết các include lên trên đầu nhé